rust/ql/test/library-tests/dataflow/sources/test_futures_io.rs

```diff
@@ -113,7 +113,7 @@ async fn test_futures_rustls_futures_io() -> io::Result<()> {
         sink(&pinned); // $ hasTaintFlow=url
         let mut cx = Context::from_waker(futures::task::noop_waker_ref());
         let buffer = pinned.poll_fill_buf(&mut cx);
-        sink(&buffer); // $ MISSING: hasTaintFlow=url
+        sink(&buffer); // $ hasTaintFlow=url
```

Comment:
I reckon this happens because `getCanonicalPath` is computed also for library structs/enums?

Reply:
I think that's the case, yes :+1: